packages/component-library/src/Collapsable/Collapsable.js

```diff
@@ -0,0 +1,77 @@
+import React, { Children } from 'react';
+import { css } from 'emotion';
+
+const toggleStyle = css`
+  padding: 10px;
+  border-bottom: none;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 10px;
+  display: block;
+  text-align: center;
+  background: rgba(238,73,80,0.4);
+`;
+
+class Collapsable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { expanded: false };
+
+    this.onToggle = this.onToggle.bind(this);
+  }
+
+  onToggle() {
+    this.setState({ expanded: !this.state.expanded });
+  }
+
+  renderToggle() {
+    const cta = (this.state.expanded ? 'Less' : 'More');
+    const arrow = (this.state.expanded ? 'up' : 'down');
+
+    return (
+      <a className={ toggleStyle } onClick={ this.onToggle }>
+       { cta }
+       <span
+        style={{ display: 'block' }}
+        className={ `fa fa-arrow-${arrow}` }>
+      </span>
+      </a>
+    );
+  }
+
+  render() {
+    const children = [];
+    let showToggle;
+    let toggle;
+
+    Children.forEach(this.props.children, (child) => {
+      if (child.props.hidden) {
+        showToggle = true;
+
+        if (this.state.expanded) {
+          children.push(child);
+        }
+      } else {
+        children.push(child);
+      }
+    });
+
+    if (showToggle) {
+      toggle = this.renderToggle();
+    }
+
+    return (
+      <div>
+        { children }
+        { toggle }
+      </div>
+    );
+  }
+}
+
+const Section = ({ children }) => children;
+
+Collapsable.Section = Section
```

Comment:
Ha, you weren't kidding when you said this was the simplest component you've ever written.